dark-node/dark_node.go

```diff
@@ -191,7 +190,10 @@ func (node *DarkNode) StartUI() {
 	})))
 
 	path := node.Config.Path
-	http.Handle("/", http.FileServer(http.Dir(path+"/ui")))
+  http.Handle("/settings", http.StripPrefix("/settings", http.FileServer(http.Dir(path + "/ui"))))
```

Comment:
Be careful about your indentation; the improvements to our automated testing will reject badly formatted code.